internal/types.go

```diff
@@ -42,6 +42,7 @@ type SyncSet struct {
 	Name        string
 	Source      json.RawMessage
 	Destination json.RawMessage
+	Disable     bool
```

Comment:
As much as I would prefer a shorter name, `disable` could have multiple connotations here. It may be best to have a more verbose name.